development/README.md

```diff
@@ -282,4 +282,191 @@ $BUILD_SCRIPTS/qgis-dev-install.sh $BUILD_DIR
 
 ## Configure/build/install QGIS in Qt Creator.app
 
-TODO
+### Install QtCreator 4.0 or better
+
+* Grab the installer from https://www.qt.io/download-open-source/
+* Run the installer. You ONLY need the qt-creator package so disable the Qt5 and don’t install Qt libs (we will use the qt4 installed by brew)
+* Once the installer is run open qt-creator
+* Make a directory in your QGIS source tree called ``build``
+* Open the top level ``CMakeLists.txt`` in QtCreator
+
+### Making a kit
+
+You need to modify the desktop kit to use your brew install qt5:
+
+``Preferences -> build and run -> Qt Versions -> Manual -> Add``
+
+Set the executable to :
+
+``/usr/local/bin/qmake``
+
+``Preferences -> build and run -> Cmake -> Manual -> Add``
+
+* **Name:** Brew CMake
+* **Path:** /usr/local/bin/cmake
+
+
+``Preferences -> build and run -> Kits -> Manual Desktop``
```

Comment:
Another reason to use the 'opt_prefix':  when you use `brew upgrade <formula>` to install the latest, but _do not_ follow that up with `brew cleanup <formula>`, as when you are just testing out the newer install. In this case, the older install is still there (and any app that references it will still work), but you do not get the benefit of the upgraded formula in the newer install prefix.